config/new_test.go

```diff
@@ -53,8 +53,10 @@ func TestNew_MockSession(t *testing.T) {
 	require := requirepkg.New(t)
 
 	// Exercise
+	var mock auth.Session
+	mock = &auth.MockSession{}
```

Comment:
This lint error is saying go prefers the more concise `mock := &auth.MockSession{}` (which both declares the variable type and its value)